style-guide/src/pages/index.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import Link from 'gatsby-link';
 
-import styles from '../styles/index.module.css';
 import fonts from '../styles/fonts.css';
+import styles from '../styles/index.module.css';
 
 export default () => {
   return (
     <div className={styles.body}>
-      <h1>Hello world</h1>
       <p>
-        <Link className={styles.link} to="/otkit-desktop-typography/">
-          link to otkit-desktop-typography
-        </Link>
+        Welcome to the OTKit Style Guide, the place to see our{' '}
```

Comment:
Why did you need to add a escaped space via `{}` here? was there something wrong with the normal breaking?